webapp/src/main/java/com/box/l10n/mojito/security/WebSecurityConfig.java

```diff
@@ -175,7 +175,8 @@ public SecurityFilterChain configure(HttpSecurity http) throws Exception {
                     "/cli/**",
                     "/js/**",
                     "/css/**",
-                    "/error")
+                    "/error",
+                    "/v3/api-docs")
```

Comment:
This endpoint should be open for open api file generation during the integration test phase